src/components/post/index.js

```diff
@@ -1,15 +1,20 @@
+import LikeIcon from "../../assets/icons/likeIcon"
+import CommentIcon from "../../assets/icons/commentIcon"
 import useModal from "../../hooks/useModal"
 import Card from "../card"
 import Comment from "../comment"
 import EditPostModal from "../editPostModal"
 import ProfileCircle from "../profileCircle"
 import "./style.css"
+import { useState } from "react"
 
-const Post = ({ name, date, content, comments = [], likes = 0 }) => {
+//likes are currently hardcoded. needs to be added on server side?
+const Post = ({ name, date, content, comments = ["Cool post"], likes }) => {
```

Comment:
Let's change this a bit. Instead of giving `comments` a default value, pass in some mock data when the component is being used. E.g.

```js
<Post 
  name={post.name}
  date={post.createdAt}
  comments={["Cool post"]}
/>
```